livereveal/main.js

```diff
@@ -106,8 +115,22 @@ function markupSlides(container) {
     
     // Put .end_space back at the end after all the rearrangement
     $('.end_space').appendTo('div#notebook-container');
+    return selected_cell_slide;
 }
 
+// Set the #slide-x-y part of the URL to control where the slideshow will start
+function setStartingSlide(selected) {
+    var start_slideshow = config.get_sync('start_slideshow_at');
+    if (start_slideshow === 'selected') {
+        // Start from the selected cell
+        window.location.hash = "/slide-"+selected[0]+"-"+selected[1];
```

Comment:
Well... that's interesting... if the behaviour is better in this way, I would not be opposed to use it... but let's make a tiny comment about that fact in the source code...
I will try to review it soon... just arrive home today :wink: